includes/data-stores/class-wc-data-store-posts.php

```diff
@@ -0,0 +1,27 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Shared logic for posts data stores.
+ *
+ * @version  2.7.0
+ * @category Class
+ * @author   WooThemes
+ */
+class WC_Data_Store_Posts  {
```

Comment:
Or maybe not even CPT, `Custom Content Type` or `Content Type` makes looks better.